docs/specifications/tests/README.md

```diff
@@ -17,10 +17,10 @@ project. All the details are in the [Master Test Plan](MasterTestPlan.md).
 
 1. Each test level has been separated into a separate directory below
    this directory.
-2. Under each test level directory there is a level.md document
+2. Under each test level directory there is a level document (README.md)
    describing the test level. See for example the level document from
-   [Syntax-TP](Syntax-TP/level.md).
-3. The level.md document describes the mapping from Test Requirement
+   [Syntax-TP](Syntax-TP/README.md).
+3. The level document (README.md) describes the mapping from Test Requirement
```

Comment:
Pointing out the filename without the directory is of little help to people who don't already know where to look. Also, it was just defined in the previous item so people can be expected to know it without further explanation.

Reply:
Here no specific directory can be given. I'll try an update.